React/game/src/app/components/Screen/Game/Square/styles.scss

```diff
@@ -0,0 +1,22 @@
+.square {
+  background: #FFF;
+  border: 1px solid #999;
+  float: left;
+  font-size: 24px;
+  font-weight: bold;
+  height: 34px;
+  line-height: 34px;
+  margin-right: -1px;
+  margin-top: -1px;
+  padding: 0;
+  text-align: center;
+  width: 34px;
+}
+
+.square:focus {
+  outline: none;
+}
+
+.kbd-navigation .square:focus {
+  background: #DDD;
```

Comment:
remember to put all the colors in a separate `colors` file

Reply:
👍